build.gradle

```diff
@@ -883,6 +884,36 @@ gradle.taskGraph.whenReady({ graph ->
   tasks.getByName('allReports').reportOn rootProject.subprojects.collect{ it.tasks.withType(Test) }.flatten()
 })
 
+task installPulse {
```

Comment:
perhaps better name is "packagePulse" since there is no system installation here